prov/verbs/src/verbs_info.c

```diff
@@ -637,8 +637,9 @@ static int fi_ibv_alloc_info(struct ibv_context *ctx, struct fi_info **info,
 			goto err;
 		}
 
-		snprintf(fi->fabric_attr->name, name_len, VERBS_IB_PREFIX "%" PRIu64,
-			 be64toh(gid.global.subnet_prefix));
+		if (snprintf(fi->fabric_attr->name, name_len, VERBS_IB_PREFIX "%" PRIx64,
+			 be64toh(gid.global.subnet_prefix)) >= name_len)
+			fi->fabric_attr->name[name_len - 1] = '\0';			 	
```

Comment:
Seems it should be:
`fi->fabric_attr->name[name_len] = '\0';`

Because we allocated `name_len + 1` elements.

Reply:
I'll double check.

Reply:
I see it now...  yep that should work.